events/hfe_events_leaders.txt

```diff
@@ -910,22 +910,6 @@ country_event = {
 	}
 	immediate = {
 		from = {
-			if = {
```

Comment:
This is important

Reply:
I checked that file. I cannot tell what you think I removed.

Reply:
OH, never mind. I see what I did. I broke the bracket system. It's fixed now.

Reply:
You removed if and else_if

Reply:
Which clause did I remove?

Reply:
Okay, I checked. IF and ELSE_IF made the two mutually exclusive. That is silly, because the Radical Diplomats can train Dolphins. A Dolphin doesn't cease to be a Dolphin just because the Kad Republic decided to "teach boat".

Reply:
wut?

Reply:
A Dolphin cannot be trained by the Kad Republic's Radical Diplomats under the clause that I removed. The two were mutually exclusive. A Dolphin doesn't cease to be a Dolphin just because the Kad Republic decided to "teach boat".